front/app/containers/UsersEditPage/ProfileForm.tsx

```diff
@@ -272,21 +247,16 @@ const ProfileForm = () => {
               label={formatMessage(messages.language)}
             />
           </SectionField>
+          <UserCustomFieldsForm
+            authenticationContext={GLOBAL_CONTEXT}
+            onChange={setExtraFormData}
+          />
+          <Box display="flex">
+            <Button type="submit" processing={methods.formState.isSubmitting}>
```

Comment:
```suggestion
            <Button processing={methods.formState.isSubmitting}>
```
Already the default.